### PR TITLE
v1.8 backports 2021-04-28

### PR DESCRIPTION
**Pull request by christarazi**
* #14492 -- Agent: Include Cilium version in output of 'cilium status --verbose' (@romanspb80)
 * #15590 -- ctmap: do not call InitMapInfo() in init() (@kkourt)
 * #15260 -- mtu: Switch to v/netlink for querying netdevs (@brb)
 * #15649 -- daemon/cmd: fix Cilium version status output (@aanm)
 * #15668 -- Fix channel panic from ipcache kvstore reconnect (@jomenxiao)
 * #15742 -- kvstore/etcd: fix etcd rate limit (QPS) not working (@ArthurChiao)
 * #15762 -- test: Skip K8sPolicy on GKE and 4.19 (@pchaigno)
 * #15804 -- pkg/k8s: reset k8s event lag metric on pod add (@aanm)
 * #15838 -- contrib: Clean output of submit-backport script (@pchaigno)
 * #15869 -- Improve the docs CRD schema version update script (@joestringer)
 * #15903 -- contrib: Ensure release tag is upstream before push (@joestringer)

Skipped due to conflicts:

 * #15622 -- Fix ethtool issues (@tklauser)
 * #15841 -- .github: remove unnecessary docker hub credentials (@aanm)
 * #15803 -- handle IP addresses modification in running nodes and CEPs (@aanm)

Once this PR is merged, you can update the PR labels via:
```upstream-prs
$ for pr in 14492 15590 15260 15649 15668 15742 15762 15804 15838 15869 15903; do contrib/backporting/set-labels.py $pr done 1.8; done
```

### CODE REVIEW

**Comment by christarazi** [comment]
test-backport-1.8

File: test/k8sT/Policies.go [hunk 1]
```diff
@@ -34,7 +34,12 @@ import (
 	v1 "k8s.io/api/core/v1"
 )
 
-var _ = Describe("K8sPolicyTest", func() {
+var _ = SkipDescribeIf(func() bool {
+	// We only need to run on 4.9 with kube-proxy and net-next with KPR
+	// and the third node. Other CI jobs are not expected to increase
+	// code coverage.
+	return helpers.RunsOnGKE() || helpers.RunsOn419Kernel()
```
Comment (pchaigno) [inline test/k8sT/Policies.go]:
```suggestion
	return helpers.IsIntegration(helpers.CIIntegrationGKE) || helpers.RunsOn419Kernel()
```

**Review by aanm — APPROVED** [review]
Looks good for my commits, thanks!

**Review by joestringer — APPROVED** [review]
Commits from me look fine :+1:

**Comment by joestringer** [comment]
Legitimate failure:

```
13:33:04  Failed to compile test:
13:33:04  
13:33:04  # github.com/cilium/cilium/test/k8sT
13:33:04  k8sT/Policies.go:41:9: undefined: helpers.RunsOnGKE
```

You can test this locally with `make -C test build`.

**Review by brb — APPROVED** [review]
My changes LGTM, thanks.

**Comment by tklauser** [comment]
> Skipped due to conflicts:
> 
>     * #15622 -- Fix ethtool issues (@tklauser)

FWIW, this depends on #15602 being merged.

**Comment by christarazi** [comment]
test-backport-1.8

**Comment by christarazi** [comment]
test-upstream-k8s

**Comment by christarazi** [comment]
test-runtime

**Review by gandro — APPROVED** [review]

**Comment by christarazi** [comment]
test-runtime

**Review by pchaigno — APPROVED** [review]
My changes look good to me :ship:

**Pull request merged**